lib/pacemaker/pcmk_options.c

```diff
@@ -39,3 +41,23 @@ pcmk__list_cluster_options(pcmk__output_t *out)
                                         pcmk__opt_context_none,
                                         pcmk__opt_list_cluster);
 }
+
+// Documented in header
+int
+pcmk_list_cluster_options(xmlNode **xml)
+{
+    pcmk__output_t *out = NULL;
+    int rc = pcmk_rc_ok;
+
+    rc = pcmk__xml_output_new(&out, xml);
+    if (rc != pcmk_rc_ok) {
+        return rc;
+    }
+
+    pcmk__register_lib_messages(out);
+
+    rc = pcmk__list_cluster_options(out);
+
+    pcmk__xml_output_finish(out, pcmk_rc2exitc(rc), xml);
+    return rc;
+}
```

Comment:
Yeah probably... maybe I was thinking about reducing the amount of doxygen lol, I dunno. Or maybe `pcmk__output_option_metadata()` started out more complex. Or -- and I'm thinking this is the one -- maybe I didn't want to have a `pcmk__output_cluster_options()` in libcrmcommon and `pcmk__list_cluster_options()` in libpacemaker because they're so similar. (The output one has to stay in libcrmcommon for the sake of the options table.)

Reply:
It would be fine to have a high-level public API in libpacemaker and rely on the libcrmcommon function internally

Reply:
That's a given at this point. I'm trying to remember why I did the enum thing in the first place, and thinking it might have been to avoid having such similar names